modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/feather.dm

```diff
@@ -8,17 +8,24 @@
 	righthand_file = 'modular_skyrat/modules/modular_items/lewd_items/icons/mob/lewd_inhands/lewd_inhand_right.dmi'
 	w_class = WEIGHT_CLASS_TINY
 
-/obj/item/tickle_feather/attack(mob/living/carbon/human/target, mob/living/carbon/human/user)
+/obj/item/tickle_feather/attack(mob/living/target, mob/living/user)
 	. = ..()
-	if(!istype(target))
+	var/mob/living/carbon/human/carbon_target
+	if(istype(target,/mob/living/carbon/human))
+		carbon_target = target
+	else if(istype(target,/mob/living/silicon/robot))
+		// Just use target var, return if it isn't human or robot
+	else
+		return
+	if(!istype(user,/mob/living/carbon/human) && !istype(user,/mob/living/silicon/robot))
 		return
 
```

Comment:
This whole file kinda sucks, but that's because of the OG code, less because of your modifications. I suppose the carbon_target checks could be replaced with istype checks, or you could make a single check `var/targetiscarbon = istype ...)`

Reply:
is `if(carbon_target)` not good? I figure it's just like a null check, but I'm not that familiar with dm